docs/data/data-grid/column-groups/column-groups.md

```diff
@@ -1,18 +1,100 @@
 ---
-title: Data Grid - Column groups 🚧
+title: Data grid - Column groups
 ---
 
-# Data grid - Column groups 🚧
+# Data grid - Column groups
 
 <p class="description">Group your columns.</p>
 
+Grouping columns allows you to have multiple levels of columns in your header and the ability, if needed, to toggle column groups to show and hide additional columns.
+
+## Define column grouping
+
+You can define the column grouping structure by providing the `columnGroupingModel` prop to the grid.
+This prop receives an array of column group.
+
+A column group is defined by at least two attributes:
```

Comment:
Should we use "property" instead of "attribute" in this context?
I would rather use "attribute" only for HTML elements

Reply:
I'm always lost with that. sounds you're right https://www.notion.so/mui-org/Technical-writing-7e55b517ac2e489a9ddb6d0f6dd765de